### PR TITLE
Prevent unauthorized players from toggling blinky plant.

### PR DESCRIPTION
**Pull request by auouymous**

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

File: mesecons_blinkyplant/init.lua [hunk 1]
```diff
@@ -32,7 +32,13 @@ mesecon.register_node("mesecons_blinkyplant:blinky_plant", {
 		fixed = {-0.3, -0.5, -0.3, 0.3, -0.5+0.7, 0.3},
 	},
 	on_timer = on_timer,
-	on_rightclick = toggle_timer,
+	on_rightclick = function(pos, node, clicker)
+		if minetest.is_protected(pos, clicker and clicker:get_player_name()) then
```
Comment (MoNTE48) [inline mesecons_blinkyplant/init.lua:36]: `clicker and clicker:get_player_name() or ""`

Comment (auouymous) [reply mesecons_blinkyplant/init.lua:36]: Can clicker even be nil, other than buggy mods that directly call it? Mesecons is the only mod I have that even checks for an entity before calling get_player_name, and none of them (including mesecons) append `or ""`.

Comment (MoNTE48) [reply mesecons_blinkyplant/init.lua:36]: Since 5.3 - sometimes it can.

Comment (auouymous) [reply mesecons_blinkyplant/init.lua]: Fixed.

**Review by sfan5 — APPROVED** [review]
Looks good.

**Comment by SmallJoker** [comment]
:+1: makes sense. Players should not influence protected circuits.

**Pull request merged**